### DISCUSSION TITLE
Shortcut to repeat all mistakes in test

### DISCUSSION THREAD

**Discussion by harshasomisetty** [monkeytypegame/monkeytype#4300] [Ideas]
Similar to how pressing tab after a test allows you to go to the next test, I'd like a shortcut to quickly practice all the missed words in a test.

**Comment by Miodec** [comment]
When on the result page, Esc > Practice words

**Reply by harshasomisetty** [reply]
I meant more of a single keyboard shortcut that would practice missed words. like pressing shift+e for Errors, or something like that. Don't want to go through a menu or click multiple buttons

**Reply by Miodec** [reply]
You can use the commandline with just your keyboard (type to search, enter to select). Not sure about adding shortcuts since it might interfere with the browser or OS.
